src/components/Garden/Metrics.js

```diff
@@ -289,67 +296,47 @@ function TokenPrice({
       <div
         css={`
           display: flex;
-          justify-content: space-between;
+          align-items: center;
+          ${textStyle('title4')};
+          margin-bottom: ${0.5 * GU}px;
         `}
       >
-        <p
-          css={`
-            ${textStyle('title4')};
-            margin-bottom: ${0.5 * GU}px;
-          `}
-        >
-          {token.symbol} Price{' '}
-          {oracleMode && (
-            <span
-              css={`
-                ${textStyle('body4')}
-              `}
-            >
-              (Oracle)
-            </span>
-          )}
-        </p>
+        {oracleText}
         {oracleMode && (
-          <Help hint="What is Oracle Price?">
-            This is the price used for funding proposals defined in stable
-            asset. Calling update price could return a reward.
-          </Help>
+          <span
+            css={`
+              padding-left: ${1 * GU}px;
+              display: inline-block;
```

Comment:
Let´s use margin instead, also the display attr seems unnecessary due to the parent div flex style
```suggestion
              margin-left: ${1 * GU}px;
```